cpp/example_code/dynamodb/Readme.md

```diff
@@ -25,12 +33,18 @@ Code excerpts that show you how to call individual service functions.
 * [Scan a table](./scan_table.cpp) (Scan)
 * [Update an item](./update_item.cpp) (UpdateItem)
 * [Update a table](./update_table.cpp) (UpdateTable)
+
 ### Scenarios
+
 Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
 * [Getting started scenario for DynamoDB](./dynamodb_getting_started_scenario.cpp) (CreateTable, PutItem, UpdateItem, BatchWriteItem, GetItem, Query, Scan, DeleteItem, DeleteTable)
+* [PartiQL scenario for DynamoDB](./dynamodb_partiql_scenario.cpp) (CreateTable, ExecuteStatement, BatchExecuteStatement, DeleteTable)
```

Comment:
Looks like there are 2 separate PartiQL scenarios in the docs, shouldn't this example list match that one? One for batches and one for single actions?